src/handlers/relabel.rs

```diff
@@ -185,7 +185,12 @@ mod tests {
         macro_rules! t {
             ($($member:ident { $($label:expr => $res:ident,)* })*) => {
                 let config = RelabelConfig {
-                    allow_unauthenticated: vec!["T-*".into(), "I-*".into(), "!I-nominated".into()],
+                    allow_unauthenticated: vec![
+                        "T-*".into(),
+                        "I-*".into(),
+                        "!I-*-nominated".into(),
```

Comment:
nit: the triagebot.toml in rust-lang/rust has `!I-*nominated`, maybe should match here.

Reply:
@Mark-Simulacrum Will do.